### ISSUE TITLE
Getting an Error : Failed to connect to yts.mx

### ISSUE THREAD

**Issue by android-bash** [KaustubhPatange/Moviesy#25]
**Please complete the following information:**
- App Version - Latest
- Affected Device(s)  - Android
 
**Describe the Bug:**

Getting an Error : Failed to connect to yts.mx

**Expected Behavior:**

Should connect and display movies list

**Comment by KaustubhPatange** [comment]
I am seeing this issue from last night & seems like the issue is from [yts.mx](https://yts.mx) website. Can you check if the website is loading in your Android Chrome browser? In my opinion, it should not!

The problem is sometimes it loads but later on, we start getting SSL issues. I've contacted the site admins (yesterday) to let them know about this behavior, lets see how long it takes for them to resolve the issue from their side.

Once this is done, the app will automatically start working normally!

**Comment by android-bash** [comment]
It opens up on browser if i use any VPN and set country to Germany etc.

**Comment by KaustubhPatange** [comment]
Yup, I know the problem is specific to the Asia! For now, you have to use VPN if you want the app to function as well or wait till YTS  owners resolve this issue.
